cpp/src/io/parquet/reader_impl_chunking_utils.cu

```diff
@@ -320,7 +320,10 @@ std::tuple<rmm::device_uvector<page_span>, size_t, size_t> compute_next_subpass(
 
   // bring back to the cpu
   auto const h_aggregated_info = cudf::detail::make_host_vector(aggregated_info, stream);
-  // print_cumulative_row_info(h_aggregated_info, "adjusted");
+
+#if defined(CHUNKING_DEBUG)
```

Comment:
Added the same preprocessing conditions around these so that if `CHUNKING_DEBUG` is defined, these prints are also automatically enabled instead of manually uncommenting them.